company-lsp.el

```diff
@@ -251,6 +251,20 @@ Return a plist of (:incomplete :candidates) if cache for PREFIX
 exists. Otherwise return nil."
   (cdr (assoc prefix company-lsp--completion-cache)))
 
+(defun company-lsp--documentation (candidate)
+  "Get the documentation from the item in the CANDIDATE.
+
+The documentation can be either string or MarkupContent. This method
+will return markdown string if it is MarkupContent, original string
+otherwise. If the documentation is not present, it will return nil
+which company can handle."
+  (let* ((item (company-lsp--candidate-item candidate))
+         (documentation (gethash "documentation" item)))
+    (if
+        (hash-table-p documentation)
```

Comment:
nit: move this line above

Reply:
The if condition should remain in the body to make the MarkupContent part clear.

Reply:
In general the condition is written on the same line of "if", but that's just a minor style issue. I'll merge this PR. Thanks